lifecycle/manager.go

```diff
@@ -138,3 +138,8 @@ func (lcs *LifecycleServicesManager) IsReleaseBundleExist(rbName, rbVersion, pro
 	rbService := lifecycle.NewReleaseBundlesService(lcs.config.GetServiceDetails(), lcs.client)
 	return rbService.ReleaseBundleExists(rbName, rbVersion, projectKey)
 }
+
+func (lcs *LifecycleServicesManager) AnnotateReleaseBundle(params lifecycle.AnnotateOperationParams) error {
```

Comment:
```suggestion
func (lcs *LifecycleServicesManager) AnnotateReleaseBundle(params lifecycle.AnnotateOperationParams, artDetails auth.ServiceDetails, client *jfroghttpclient.JfrogHttpClient) error {
```
this method can define `NewAnnonateOPeration{}` with rtDetails, client etc so all required details are available in a single receiver instead of passing params

Reply:
@bhanurp I am not sure.